privacyidea/models.py

```diff
@@ -482,14 +485,13 @@ def get(self, key=None, fallback=None, save=False):
         if key is None:
             return self.get_vars(save=save)
 
-        if hasattr(self, key):
-            return getattr(self, key)
-        else:
-            return fallback
-    
+        td = self.get_vars(save=save)
+        return td.get(key, fallback)
+
     @log_with(log)
     def get_vars(self, save=False):
         log.debug('get_vars()')
+        tokenowner = self.owners.first()
```

Comment:
I think ``token.owners.first()`` is used at so many places that it would be warranted to add a property to ``Token`` named ``first_owner``, which could be used instead. What do you think?

Reply:
Done in 65cf90e8